events/isc-fall-2017.md

```diff
@@ -39,13 +62,30 @@ TBA
 
 ### Area Highlights
 
-TBA
+The [city of Naperville](http://www.naperville.il.us/) is a suburb of Chicago, located about forty minutes west of the Chicago downtown and about forty minutes away from the O'Hare International Airport. It was ranked among the nation's safest cities by USAToday and Business Insider. Naperville was voted the second-best place to live in the United States by Money magazine in 2006. It was rated 1st on the list of best cities for early retirement in 2013 by Kiplinger. As of the 2010 census, the city had a population of 141,853, which was estimated to have increased to 147,100 by July 2015. It is the fifth-largest city in Illinois.
+
+Downtown Naperville features [its own riverwalk](http://www.naperville.il.us/enjoy-naperville/naperville-riverwalk/), [created in 1981 to celebrate the city's Sesquicentennial](http://www.napervilleparks.org/the-riverwalk) (150th anniversary). The park features covered bridges, relaxing fountains and distinctive shepherd's crook lights. 
+
+
+<img width="600" src="/InnerSourceCommons/assets/img/naperville_riverwalk_4755340966_fc7390571a_z_brian_jelonek.jpg"/>
+<p style="font-size: 8pt; font-style: italic;">Photo by <a href="https://www.flickr.com/photos/jelonekphotography/4755340966/">Brian Jelonek</a></p>
+<img width="600" src="/InnerSourceCommons/assets/img/naperville_riverwalk_4754699889_719ac628ba_z_brian_jelonek.jpg"/>
```

Comment:
During build time, HTML-Proofer is complaining that this image does not have an alt attribute.